README.md

```diff
@@ -44,6 +44,9 @@ run style checks use `composer cs`.
 
 ## Release notes
 
+### Next Version (2.0.x)
+* Ability ro re-ruse the file `cli-config.php` when including `FundraisingStore` in an application is removed.
```

Comment:
Sounds like a breaking change rather than a bugfix, so this is not semver compliant. Or can is reasonably be seen as a bugfix, ie something we never thought clients should rely upon?

I'm not saying we can't deviate from semver, just that it is what this seems to do